src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {hide as hideSidebar} from '../../../../libs/actions/Sidebar';
+import withWindowDimensions, {
+    windowDimensionsPropTypes,
+} from '../../../../components/withWindowDimensions';
+import {getNewChatOptions} from '../../../../libs/OptionsListUtils';
+
+import OptionsSelector from '../../../../components/OptionsSelector';
+import ONYXKEYS from '../../../../ONYXKEYS';
+
+const personalDetailsPropTypes = PropTypes.shape({
```

Comment:
NAB but I don't think this needs to be declared separately from the rest of the prop types

Reply:
My thought process behind this was to implement a consistent PropType declaration with `NewChatPage` in #1419. Should I merge it into a single declaration regardless?

Reply:
I think this is fine, as it also matches our existing usages.